src/SFA.DAS.EAS.Support.Web/Controllers/AccountController.cs

```diff
@@ -119,14 +145,20 @@ public async Task<ActionResult> Finance(string id)
                 return View(vm);
             }
 
+            _log.Info($"AccountController-Finance : HttpNotFound for id : {id} ");
+
             return HttpNotFound();
         }
 
         [Route("account/levysubmissions/{id}/{payeSchemeId}")]
         public async Task<ActionResult> PayeSchemeLevySubmissions(string id, string payeSchemeId)
         {
+            _log.Info($"AccountController-PayeSchemeLevySubmissions : Getting PayeLevySubmissionsResponseCodes Response for id : {id}");
+
             var response = await _payeLevySubmissionsHandler.FindPayeSchemeLevySubmissions(id, payeSchemeId);
 
+            _log.Info($"AccountController-PayeSchemeLevySubmissions : PayeLevySubmissionsResponseCodes Response : {response}");
```

Comment:
logging the response won't log any of the response data

Reply:
Removed the logs as we discussed.